lib/helpers.py

```diff
@@ -423,7 +436,7 @@ def _merge(marks_df, attributions_df):
 
 
 class _CSVHelpers(object):
-    def __init__(self, customer, revenue_event, chunk_size=10 ** 6):
+    def __init__(self, customer, revenue_event, chunk_size=10 ** 6, export_user_ids=False):
```

Comment:
Judging from where and how this flag is used, there are two options:

1. Keep the flag as `_CSVHelpers` class property, but then remove it from methods arguments (some of them have to become non-static because of that, but it's fine)
2. Remove it from `_CSVHelpers` class properties and add as `read_csv` method argument and keep it as an argument to the methods.

Either option would be fine and clean, while now it's something in between the two, and it looks quite a bit odd and inconsistent.

Reply:
I keep it as a class property now